requirements/deploy.txt

```diff
@@ -121,7 +121,7 @@ django==4.2.16
     #   django-timezone-field
     #   djangorestframework
     #   jsonfield
-django-allauth[mfa,saml,socialaccount]==64.2.1
+django-allauth[mfa,saml,socialaccount]==65.0.2
```

Comment:
Important changes:

> When the user is partially logged in (e.g. pending 2FA, or login by code), accessing the login/signup page now redirects to the pending login stage. This is similar to the redirect that was already in place when the user was fully authenticated while accessing the login/signup page. As a result, cancelling (logging out of) the pending stage requires an actual logout POST instead of merely linking back to e.g. the login page. The builtin templates handle this change transparently, but if you copied any of the templates involving the login stages you will have to adjust the cancel link into a logout POST.
 
cc @agjohnson since you worked on the templates.
cc @stsewd since you implemented 2FA.

Reply:
Looks like templates may need to be updated. Here is the PR https://codeberg.org/allauth/django-allauth/pulls/4111/files.